File: omeskiosk/Binary/Classes/TicketLayer/Grup.php
<?php
/*
-- =============================================
-- Author:		EKOMURCU
-- Create date: 01.11.2017
-- Description:	Grup Sınıfı
-- ============================================= 
 */
class Grup
{
		protected function IsGroupInLunchBreak()
        {
            $dtTicketRequest = date('H:i:s');
            if (!($dtTicketRequest > $this->OgleArasiBaslangic && $dtTicketRequest < $this->OgleArasiBitis))
            {
                return false;
            }
            else
            {
                return true;
            }
        }
		 protected function IsGroupOutOfWorkingHours()
        {
            $dtTicketRequest = date('H:i:s');
            if (!($dtTicketRequest > $this->MesaiBaslangic && $dtTicketRequest < $this->MesaiBitis))
            {
                return false;
            }
            else
            {
                return true;
            }
        }
}
?>

File: omesLCD/QPU_SerialPort 09 10 2017/Classes/SerialPort/TicketLayer/Grup.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace QPU_SerialPort.Classes.TicketLayer {
    public partial class Grup {
        #region  Members/Propertieses
                                        public bool GrupOgleTatilinde { get; set; }
                                        public bool GrupMesaiSaatiDisinda { get; set; }
                                        public DateTime GrupBiletIstekTarihi { get; set; }
        #endregion

        #region Logical Process Methods
                                                                
                                        public bool IsGroupInLunchBreak() {
            if (!(Bilet.newBilet.BiletTarih > OgleArasiBaslangic && Bilet.newBilet.BiletTarih < OgleArasiBitis)) {
                                return false;
            }
            else {                 return true;
            }
        }
                                                private bool IsGroupOutOfWorkingHours() {
            if (!(Bilet.newBilet.BiletTarih > MesaiBaslangic && Bilet.newBilet.BiletTarih < MesaiBitis)) {
                                return false;             }
            else {                 return true;
            }
        }

        #endregion
    }
}
